execution_engine_testing/tests/src/test/regression/host_function_metrics_size_and_gas_cost.rs

```diff
@@ -65,6 +65,10 @@ fn host_function_metrics_has_acceptable_size() {
         "Performance improvement: contract host-function-metrics became only {} bytes long; please adjust this regression test.",
         size
     );
+    println!(
+        "contract host-function-metrics byte size: {}, ubound: {}",
+        size, HOST_FUNCTION_METRICS_MAX_SIZE
+    )
```

Comment:
nit: This seems to be a remnant of another test and it's of no use in `dev` because the print happens after the asserts, so we wouldn't get to it if the test failed anyway.